packages/story-editor/src/masks/display.js

```diff
@@ -85,28 +107,53 @@ export default function WithMask({
     previewMode ? '-preview' : ''
   }-${randomId}`;
 
+  const wrapperStyle = !showSingleBorder
+    ? null
+    : getBorderWrapperStyle(maskId, borderWidth * 2);
+
   return (
-    <div
-      style={{
-        pointerEvents: 'initial',
-        ...fullStyle,
-        ...(!isBackground ? { clipPath: `url(#${maskId})` } : {}),
-      }}
-      {...rest}
-    >
-      <svg width={0} height={0}>
-        <defs>
-          <clipPath
-            id={maskId}
-            transform={`scale(1 ${mask.ratio})`}
-            clipPathUnits="objectBoundingBox"
+    <>
+      {showSingleBorder && (
+        <div style={{ ...style, ...wrapperStyle }}>
```

Comment:
Is there difference between the `display` and `output` for displaying the SVG border? If not, then maybe we could even move this to an util as well, e.g. `getMaskBorder` or something like that. WDYT?

Reply:
yeah, no difference really so moving to a util would eliminate some duplication